### PR TITLE
chore(catalog): upgrade meta data for catalog

### PR DESCRIPTION
**Pull request by danielgolden**
### Upgrade the project meta data to support new catalog commands
The NR1 CLI will soon be updated with commands that enable one to request to have their app added to the catalog. In order to prepare the existing open source NR1 apps for that coming change I've added meta data according to what the catalog commands will be looking for.

- Add a `config.json` file to a `catalog` directory in the repo root, [matching this format](https://github.com/newrelic/nr1-catalog/blob/master/examples/catalog/config.json).
- Move the `/screenshots` directory into `catalog`:
    - Update the screenshot file names ([to match this format](https://github.com/newrelic/nr1-catalog/blob/master/examples/catalog/documentation.md))
    - Update the screenshot links in the `README.md`.
- Add [the new header image](https://newrelic.slack.com/archives/GNR5YH7HP/p1583418095011300) to the top of `README.md`.
- Add a documentation.md file to a catalog directory in the repo root, [matching this format](https://github.com/newrelic/nr1-catalog/blob/master/examples/catalog/documentation.md).
    - Update the relative paths in `documentation.md` to be absolute paths
- (Optional) If there's additional documentation referenced in the repo, add a `additionalInfo.md` file to a `catalog` directory in the repo root, [matching this format](https://github.com/newrelic/nr1-catalog/blob/master/examples/catalog/additionalInfo.md).
- Ensure the `README.md` commands section matches [Justin's snippet](https://newrelic.slack.com/archives/GNR5YH7HP/p1583420716020000).
- Run `npm start` and let the CLI move the `nr1.json`
    - Edit that file to make sure the name and description make sense.

---

Resolves newrelic#32

### CODE REVIEW

File: catalog/documentation.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+Open a command prompt in the nerdpack's directory and run the following commands.
+
+```bash
+# If you need to create a new uuid for the account to which you're deploying this Nerdpack, use the following
+# nr1 nerdpack:uuid -g [--profile=your_profile_name]
+# to see a list of APIkeys / profiles available in your development environment, run nr1 profiles:list
+nr1 nerdpack:publish [--profile=your_profile_name]
+nr1 nerdpack:deploy [-c [DEV|BETA|STABLE]] [--profile=your_profile_name]
+nr1 nerdpack:subscribe [-c [DEV|BETA|STABLE]] [--profile=your_profile_name]
+```
```
Comment (jbeveland27) [inline catalog/documentation.md]: Looks like this missed being updated both here and in README.md.

File: catalog/config.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "tagline": "Identify right-sizing opportunities and potential savings of your AWS, GCP, and Azure instances across your cloud environment",
+  "repository": "https://github.com/newrelic/nr1-cloud-optimize.git",
+  "details": "Cloud Optimize analyzes your cloud environment using the New Relic Infrastructure cloud integrations. \n The application compares the size of your instances to their utilization, identifying resources that are sized larger than needed. Cloud Optimize will estimate your savings by optimizing resource size.\n Users of Cloud Optimize are able to select the hosts, regions and other configurations to specify their unique business use cases.\n This application supports AWS, GCP, and Azure cloud infrastructures.",
```
Comment (jbeveland27) [inline catalog/config.json]: I think putting two `\n\n` will create the space between paragraphs that we're looking for, but I might have that wrong (I used `\n\n` in the PRs I did, but maybe I need to review as well). We should probably confirm with devex team to be sure.

File: catalog/additionalInfo.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+## Additional Info
```
Comment (jbeveland27) [inline catalog/additionalInfo.md]: This file should be removed, as this placeholder content will display if it's here.

**Comment by danielgolden** [comment]
Updated each of these. Can you give this one more pass @jbeveland27?

**Comment by danielgolden** [comment]
Just going to merge in the spirit of... "I think I got it all"

**Pull request merged**